tests/integration/components/bs-button-test.js

```diff
@@ -1,19 +1,48 @@
 import { moduleForComponent } from 'ember-qunit';
-import test from 'ember-sinon-qunit/test-support/test';
+import test from '../../helpers/bootstrap-test';
 import hbs from 'htmlbars-inline-precompile';
 import Ember from 'ember';
 
 moduleForComponent('bs-button', 'Integration | Component | bs-button', {
   integration: true
 });
 
-test('button has correct CSS classes', function(assert) {
-  this.render(hbs`{{#bs-button type="primary" size="lg" active=true block=true}}Test{{/bs-button}}`);
+test(3, 'button has correct default markup', function(assert) {
```

Comment:
If you looked at the tests without being familiar with the way we've been thinking, this would look mysterious. I think it would make less of a learning curve for potential contributors if we spell it out.